client/src/App.js

```diff
@@ -1,13 +1,1568 @@
+import React, { useState, useEffect } from "react";
 import "./App.css";
+import VideoComponent from "./VideoComponent";
```

Comment:
In a React application, it's generally not a good practice to add lots of functions directly in the `App.js` file which are not going to be re-used across other components. 
The App.js file should primarily focus on rendering the main components and managing the state of your application. 
I'm referring to `handlers` functions